applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1943,6 +1887,37 @@ Vector UPwSmallStrainElement<TDim, TNumNodes>::GetPressureSolutionVector()
     return result;
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+void UPwSmallStrainElement<TDim, TNumNodes>::CalculateAnyOfMaterialResponse(
```

Comment:
I know that we have two identical functions now for small strain and for diff order, but I still think in total we remove more duplication than we added and the duplication is a lot more localized now.